package.json

```diff
@@ -8,9 +7,14 @@
   "es2015": "./esm2015/ngx-tabs-libdemo.js",
   "typings": "./ngx-tabs-libdemo.d.ts",
   "scripts": {
-    "build": "ng-packagr -p package.json",
+    "build": "bazel build //...",
     "copyPackageJson": "copyfiles package.json dist",
-    "semantic-release": "semantic-release pre && npm run copyPackageJson && npm publish dist && semantic-release post"
+    "semantic-release": "semantic-release pre && npm run copyPackageJson && npm publish dist && semantic-release post",
+    "postinstall": "ngc -p angular.tsconfig.json",
+    "preskylint": "bazel build --noshow_progress @io_bazel//src/tools/skylark/java/com/google/devtools/skylark/skylint:Skylint",
+    "skylint": "find . -type f \\( -name BUILD -or -name BUILD.bazel -or -name *.bzl \\) ! -path \"*/node_modules/*\" ! -path \"./dist/*\" | xargs $(bazel info bazel-bin)/external/io_bazel/src/tools/skylark/java/com/google/devtools/skylark/skylint/Skylint",
```

Comment:
What is skylint?

Reply:
Oh, found something: https://docs.bazel.build/versions/master/skylark/skylint.html

Reply:
sadly package.json doesn't permit comments unless you use the wacky trick

```
{
  "skylint": "this is a poor mans comment",
  "skylint": "find . -type f \..."
}
```

so not sure what to improve here. I could name the tasks "bazel-lint" and "bazel-format", maybe that would help? But then does the user need to know what the underlying tools are called? Maybe not. Let's try that.

Reply:
Yes it would probably be easier to immediately get the meaning of the command. The tool eventually can be extracted from the command instruction in case